src/Model/SimplePayment.php

```diff
@@ -132,6 +137,7 @@ public function __construct($values)
         $this->validTo = new DateTime($data['valid_to']);
         $this->fee = $data['fee'];
         $this->description = $data['description'];
+        $this->description_for_merchant = $data['description_for_merchant'];
```

Comment:
This is break change, it is somethink like new required parameter. If some one use `SimplePayment` contructor for some reason, without this key his code will crash.
Add some check if `description_for_merchant` key realy exist in input array.